README.md

```diff
@@ -1,24 +1,74 @@
-# README
+# Accentor API
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+API for Accentor, a modern music server focusing on metadata.
 
-Things you may want to cover:
+## Why use Accentor?
+Accentor gives you complete control over your music. You can build
+your own collection (with good old CD's, bandcamp downloads, ...) in
+the sound quality that you want and stream it either through the [web
+frontend](https://github.com/accentor/web) or [android
+app](https://github.com/accentor/android).
 
-* Ruby version
+Accentor is focused on metadata. We allow you to add detailed metadata
+to your music collection, beyond what the tags inside an audio file
+are capable of. Album and tracks can have multiple artists with a
+different name on different albums/tracks, albums can have multiple
+labels and tracks can have multiple genres.
 
-* System dependencies
+The metadata is completely in your control: you can edit it however
+you want.
 
-* Configuration
+## How to deploy
 
-* Database creation
+1. Make sure you have the correct dependencies installed. This is
+   mostly ffmpeg and ruby.
+1. Create a checkout of the code.
+1. Install all ruby dependencies with `bundle install`.
+1. Setup the database. Only postgresql is supported. Don't forget to
+   run `rails db:setup`.
+1. Make sure to perform migrations before starting the server. Also
+   run the `ffmpeg:check_version` rake task.
+1. Run `puma -C config/puma.rb` to start the server. You probably want
+   to set the following environment variables:
+    * DATABASE_URL
```

Comment:
I would split this list between those that should be set, and the optional ones. the DATABASE_URL and RAILS_ENV=production seem non-optional to me